packages/puppeteer-crawler/src/internals/enqueue-links/click-elements.ts

```diff
@@ -46,7 +47,7 @@ export interface EnqueueLinksByClickingElementsOptions {
     /**
      * A request queue to which the URLs will be enqueued.
      */
-    requestQueue: RequestQueue;
+    requestQueue: RequestQueue | RequestQueueV2;
```

Comment:
this change is very repetitive and given there are literally no other changes in the basic crawler, it feels like those interfaces are already matching each other (so things should work fine without this)

would be better to have a common interface that both classes implement

Reply:
Will be handled once I work on deduplicating the interfaces